<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuiMainWindow</class>
 <widget class="QMainWindow" name="GuiMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidgetMain">
      <widget class="QWidget" name="widgetEmpty">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="labelLogo">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="rsrc.qrc">:/pics/logo.png</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="MACHWidget" name="widgetMACHO"/>
      <widget class="MACHOFATWidget" name="widgetMACHOFAT"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>835</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionDemangle"/>
    <addaction name="actionShortcuts"/>
    <addaction name="actionOptions"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionShortcuts">
   <property name="text">
    <string>Shortcuts</string>
   </property>
  </action>
  <action name="actionDemangle">
   <property name="text">
    <string>Demangle</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MACHWidget</class>
   <extends>QWidget</extends>
   <header location="global">machwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MACHOFATWidget</class>
   <extends>QWidget</extends>
   <header>machofatwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="rsrc.qrc"/>
 </resources>
 <connections/>
</ui>
